win32.js

```diff
@@ -16,15 +19,20 @@ const spawnOpts = {
 // DefaultIPGateway             GatewayCostMetric  Index  IPConnectionMetric
 // {"1.2.3.4", "2001:db8::1"}   {0, 256}           12     25
 // {"2.3.4.5"}                  {25}               12     55
-function parseGwTable(gwTable, family) {
+function parseGwTable(gwTable, family, command) {
   let [bestGw, bestMetric, bestId] = [null, null, null];
 
   for (let line of (gwTable || "").trim().split(/\r?\n/).splice(1)) {
     line = line.trim();
     const [_, gwArr, gwCostsArr, id, ipMetric] = /({.+?}) +?({.+?}) +?([0-9]+) +?([0-9]+)/g.exec(line) || [];
     if (!gwArr) continue;
 
-    const gateways = (gwArr.match(/"(.+?)"/g) || []).map(match => match.substring(1, match.length - 1));
+    let finalGwArr = gwArr;
+    if (command === "powershell") {
+      finalGwArr = `{ ${gwArr.replace(/[{}]/g, "").split(", ").map((value) => `"${value}"`).join(", ")} }`;
```

Comment:
Powershell table doesn't show IP's between `" "`. Added it for powershell only. I can't find a better way.